pkg/app/piped/platformprovider/kubernetes/kustomize_test.go

```diff
@@ -36,11 +36,36 @@ func TestKustomizeTemplate(t *testing.T) {
 
 	kustomizePath, _, err := toolregistry.DefaultRegistry().Kustomize(ctx, "")
 	require.NoError(t, err)
+	helmPath, _, err := toolregistry.DefaultRegistry().Helm(ctx, "")
```

Comment:
Please check the error is nil?
```suggestion
	helmPath, _, err := toolregistry.DefaultRegistry().Helm(ctx, "")
	require.NoError(t, err)
```

Reply:
Ugh, fixed in [`e8860e1` (#5636)](https://github.com/pipe-cd/pipecd/pull/5636/commits/e8860e1df88ab4271cbbfcdbc9ede76456b628af)!